include/CL/sycl/vec.hpp

```diff
@@ -143,6 +143,72 @@ class vec : public detail::small_array<DataType,
     return std::tuple_cat(flatten<V>(i)...);
   }
 
+public:
+  /** An accessor to the first variable of a small array
```

Comment:
I guess we need to replace `small array` by `vec` or `vector` everywhere, whether we talk about the class or the concept.

Reply:
I was thinking not only about the code but also about the comments everywhere.